sound/soc/codecs/hdac_hda.c

```diff
@@ -603,12 +607,10 @@ static int hdac_hda_dev_probe(struct hdac_device *hdev)
 
 static int hdac_hda_dev_remove(struct hdac_device *hdev)
 {
-	struct hdac_hda_priv *hda_pvt;
-
-	hda_pvt = dev_get_drvdata(&hdev->dev);
-	if (hda_pvt && hda_pvt->codec.registered)
-		cancel_delayed_work_sync(&hda_pvt->codec.jackpoll_work);
-
+	/*
+	 * Resources are freed hdac_hda_codec_remove(). This
```

Comment:
only minor , a missing "in"

Reply:
@kv2019i can you add a fixup! PR for this and merge it directly? no point in everyone reviewing this.

Reply:
Doh, sorry @ranj063 and @plbossart , I missed this. I'll make a fixup.

Reply:
Tackled in https://github.com/thesofproject/linux/pull/2262